gitpod-io/gitpod
# [dashboard] long term caching of icons/images

#### What it does

Process all images via webpack for proper caching.

#### How to test

- Check that images are properly displayed.
- Open dev tools, make sure caching is enabled and reload the page, images should be loaded in parallel under 10ms.
